content/vault/v1.21.x/content/docs/updates/important-changes.mdx

```diff
@@ -81,3 +81,59 @@ filters you have two options:
 
 1. Spread them out among the nodes of the Vault cluster.
 1. Only subscribe to events on the active node of the cluster.
+
+### GUI KV v2 metadata list request fails for some policies ((#gui-kvv2-metadata-policy))
+
+| Change      | Affected version | Fixed version
+| ----------- | ---------------- | -------------
+| Known issue | 1.21.0           | None
+
+#### Issue
+
+Users cannot list KV v2 secrets in the GUI if their policy grants `list` access to metadata 
+but the policy path does not include a trailing slash.
+
+Example of a policy that previously granted users list access and now returns an error:
+
+```
+path "secret/metadata/:path" {   
+  capabilities = ["list"] 
+}
+```
+
+#### Workaround
+
+To resolve the policy issue, add a trailing slash to the policy path until the fix is released:
```

Comment:
```suggestion
Option 1: Add a trailing slash to the policy path:
```
We try to avoid hand-waving at future behavior/fixes.

Reply:
yeah, i'm not sure this is a recommended solution as it is, so I think I'll swap it to `option 2` as well